### PR TITLE
add some notes on using rslog

### PR DESCRIPTION
**Pull request by msarahan**
Thanks @jonyoder and @marcosnav for comments on #97. Please let me know if I stated anything here incorrectly or unclearly.

### CODE REVIEW

File: pkg/rslog/README.md
Comment (jonyoder) [inline pkg/rslog/README.md:60]: It looks like this needs indentation.

File: pkg/rslog/README.md
Comment (jonyoder) [inline pkg/rslog/README.md:6]: This is a public repo. Should we reference an internal Confluence page? It may be useful for internal users, but it's also unfortunate for those with no access. Maybe we should consider adding another README here that summarizes the standards.

File: pkg/rslog/README.md
Comment (jonyoder) [inline pkg/rslog/README.md:42]: Here, and below, consider using a more generic path that does not include `rstudio`.

File: pkg/rslog/README.md
Comment (jonyoder) [inline pkg/rslog/README.md:75]: Instead of saying "in multiple files", consider something like:
> You can pass around this `compLogger` as you would any other logger that implements the `rslog.Logger` interface, but it does not replace the default logger singleton. If you would like to instantiate the singleton as a custom logger, you need to change the rslog default logger with a factory:

File: pkg/rslog/README.md
Comment (jonyoder) [inline pkg/rslog/README.md:19]: > There should be a single log file per service-child process logs should be read into the parent to be put into the...

Should this be two sentences?

**Review by jonyoder — APPROVED** [review]

File: pkg/rslog/README.md
Comment (msarahan) [reply pkg/rslog/README.md:19]: Yeah... clumsy editing. Thanks for catching that.

**Comment by msarahan** [comment]
Thanks @jonyoder for the improvements. I'm in no rush to merge this, so I'll wait until @marcosnav has time to approve or improve this before merging.

**Review by marcosnav — APPROVED** [review]
Looks good! Thanks for this @msarahan !

**Pull request merged**
